src/react-apps/ux-editor/src/actions/formFillerActions/actions/updateFormData/index.ts

```diff
@@ -5,12 +5,14 @@ export interface IUpdateFormDataAction extends Action {
   formData: any;
   componentID: string;
   dataModelElement: IDataModelFieldElement;
+  dataModelBinding?: string;
+  validate?: boolean;
```

Comment:
Validate what?

Reply:
Do we really need the validate-boolean? Do we want unvalidated data in your formData?

Reply:
Removed, see comment under.